scripts/imgtool/image.py

```diff
@@ -81,21 +94,30 @@ def load(cls, path, included_header=False, **kwargs):
         obj.check()
         return obj
 
-    def __init__(self, version=None, header_size=IMAGE_HEADER_SIZE, pad=0):
+    def __init__(self, version=None, header_size=IMAGE_HEADER_SIZE, pad=0,
+                 hex_base_addr=None):
```

Comment:
Why is this `hex_base_addr`? You might imagine loading the base address from an ELF image too. Is there a reason for this not to just be `base_addr`, and have `load()` fill it in only for `hex` right now?

Reply:
Agreed!